### DISCUSSION TITLE
2008/01/03/%E5%88%B6%E4%BD%9C%E4%B8%8D%E8%A7%84%E5%88%99%E5%9B%BE%E5%BD%A2%E7%9A%84%E5%AD%90vi%E5%9B%BE%E6%A0%87

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#202] [Announcements]
# 2008/01/03/%E5%88%B6%E4%BD%9C%E4%B8%8D%E8%A7%84%E5%88%99%E5%9B%BE%E5%BD%A2%E7%9A%84%E5%AD%90vi%E5%9B%BE%E6%A0%87

大多数子 VI 的图标都是一个正方块。但有时候，为了程序代码美观、易读，需要把子 VI 作成不规则图形显示在调用它的父 VI 代码上。LabVIEW 中很多自带的函数采用的就不是32×32的正方形，比如说加减乘除运算函数等。我们已可以把子 VI 作成非正方形方块，比如图1中的 My Function.vi，我把它的图标做成了三角形。

https://blog.qizhen.xyz/2008/01/03/%E5%88%B6%E4%BD%9C%E4%B8%8D%E8%A7%84%E5%88%99%E5%9B%BE%E5%BD%A2%E7%9A%84%E5%AD%90vi%E5%9B%BE%E6%A0%87

**Comment by ruanqizhen** [comment]
之前博客下的留言：

ruanqizhen
2011年03月23日 10:50 编辑
我对你问的问题没有什么研究，不过听起来你的问题更像是曲线吧。
如果你的图形很复杂，可以试试看用LabVIEW的3D picture control画。

回复
wsmaggie512
2011年03月22日 11:17 编辑
hello ,i want to ask you some problem about of labview, please instruct a guide,thank you !
我想用labview画个立体图，在xy平面上一条曲线，表示的是船走的航线，在船停留的那一处，z轴上也有一相应数据，表示的是水深，我想在画航线的同时，也能画出相应的水深，而在航线曲线上，能否画出切线代表水的流速？流速的大小可以使航行线有宽有窄。这个用labview的三维曲面还是三维曲线画比较好画？用曲线的话，显示控件上能否同时显示面和线？曲线之间能否进行颜色填充，让画面更直观一些？还是用matlab画图，然后用labview调用更简单一些？很多问题网上查不到相应的资料，希望您能给我指点一下，非常感谢~！

回复
